src/contracts/Config.ts

```diff
@@ -2,7 +2,7 @@ import { AxiosRequestConfig } from 'axios'
 import { Model } from '@vuex-orm/core'
 
 export interface Config extends AxiosRequestConfig {
-  dataKey?: string
+  dataTransformer?: ((response: object) => object),
```

Comment:
Maybe the argument could be `data` and `response`? I think most of the time users care mainly about `data` property. And sometimes, they could use whole `response` object to check for example header value or something.

```ts
dataTransformer?: (data: any, response: AxiosResponse) => object
```

Reply:
Hum I agree that most people care about data!

In the other hand, data is part of the response and we should encourage using destructuration to access it to keep the method signature as simple as possible.

For instance:

```js
    dataTransformer: ({ data, headers }) => {
        // Do stuff with data and headers
       // Do stuff with data.data
    }
```

What do you think?

ps: I will use the `AxiosResponse` as type, thanks!

Reply:
I think it's better now, what do you think ?

Reply:
Oh well, why didn't I though about destructuration? You're so correct. Yes we should take this path instead of separating the argument! Thanks a lot!

Please take a look at [this comment](https://github.com/vuex-orm/plugin-axios/pull/80#pullrequestreview-308269889) as well!

Reply:
Great! 

Oups didn't see this one. Will do it today, thank you !

Reply:
Thank you so much! 🤝